frontend/TvCv19.Frontend/ClientApp/projects/shared-caregiver/src/lib/carer-menu/carer-menu.component.html

```diff
@@ -0,0 +1,50 @@
+<div class="main-menu-text"><p>Menu</p></div>
+
+<div class="menu">
+  <mat-action-list>
+    <div class="menu__item" mat-list-item routerLink="../patients">
+      <span class="menu__header">Patients</span>
+      <span>List of Patients</span>
+    </div>
+    <div class="menu__item" mat-list-item routerLink="./">
+      <span class="menu__header">Notifications</span>
+      <span>Escalated Cases</span>
+    </div>
+    <div class="menu__item" mat-list-item routerLink="../change-shift">
+      <span class="menu__header">Shift Change</span>
+      <span>Hand off Patients</span>
+    </div>
+    <div class="menu__item" mat-list-item (click)="toggleQr()">
+      <!-- routerLink="../../2/patients"> -->
+      <span class="menu__header">QR Code</span>
+      <span>Display the QR code</span>
+    </div>
+    <div *ngIf="showQR" class="physician">
+      <div class="physician__qr" routerLink="../physician/{{ physician.id }}">
```

Comment:
you can reduce most of this code just down to the qrcode element as there is no need to link or display the physicians name

Reply:
updated